generate-status/write_status.py

```diff
@@ -0,0 +1,44 @@
+import json
+import datetime
+
+subject = context["repositories"]
+
+status = {
+  "overall_status": "",
+  "messages": [],
+  "subsystems": []
+}
+ocp_subsystem = {
+  "name": "openshift",
+  "status": "",
+  "state": "",
+  "info": "",
+  "updated": "",
+  "access_urls": [],
+  "messages": []
+}
+
+with open(f"../../{subject['directory']}/engagement.json", "r") as read_file:
+  engagement = json.load(read_file)
+
+current_state = subject["anarchy_subject"]["spec"]["vars"]["current_state"]
+desired_state = subject["anarchy_subject"]["spec"]["vars"]["desired_state"]
+
+ocp_subsystem["status"] = "green" if current_state == desired_state else "yellow"
+ocp_subsystem["state"] = current_state
+ocp_subsystem["info"] = "Working as expected" if current_state == desired_state else "Contact SRE team"
```

Comment:
I wonder if could/should add a 3rd option here for when the `state` is `provisioning` (IIRC this is that value of `current_state` during  provisioning). My concern is that while the infrastructure is being provisioned the `status` will be `yellow` and the `info` will say `Contact SRE team`.
I'm also not overly concerned by this so happy to merge as is and we can update in the future if this turns out to be a real issue.